playbooks/roles/edxapp/tasks/main.yml

```diff
@@ -1,44 +1,48 @@
-
-- name: Configure Nginx SSL
-  sudo: yes
-  shell: bash ./configure_ssl.sh {{ NGINX_ENABLE_SSL }} {{ EDXAPP_LMS_SSL_NGINX_PORT }} {{ EDXAPP_CMS_SSL_NGINX_FRONTEND_PORT }}
-  args:
-    chdir: "{{ edxapp_files_path }}/nginx"
-  register: bash
-
-# Print the shell tasks's stdout and stderr.
-- debug: var="bash.stdout_lines + [ bash.stderr ]"
-
-- name: Restart Nginx to use the new settings
-  sudo: yes
-  command: service nginx restart
-
-#- name: Apply patch to authenticate user after their reset confirmation goes through
-#  sudo: yes
-#  shell: patch -p1 -d {{ edx_platform_path }} < {{ edxapp_files_path }}/patches/9b1f89d19ad26625859f887b12931f4d48dd3ff0_updated.patch
-#  register: patch_result
-#  ignore_errors: True
-
-# Print the results from patching.
-#- debug: var="patch_result.stdout_lines + [ patch_result.stderr ]"
-
-# Patch fails on stamp - re-enable once fixed
-#- name: Recompile python files touched by patching
-#  sudo: yes
-#  shell: python -m py_compile {{ forms_py }} {{ test_password_policy_py }} {{ test_reset_password_py }} {{ views_py }} {{ password_policy_validators_py }} {{ test_password_history_py }}
-#  register: compile_result
-#  when: patch_result|success
-
-# Print the results from compiling.
-#- debug: var="compile_result.stdout_lines + [ compile_result.stderr ]"
-#  when: patch_result|success
+---
+- name: Apply theming compile assets workaround
+  sudo: yes
+  shell: bash ./update_theme.sh
+  args:
+    chdir: "{{ edxapp_files_path }}"
+  register: bash_theme
+
+# Print the shell tasks's stdout and stderr.
+- debug: var="bash_theme.stdout_lines + [ bash_theme.stderr ]"
+
+- name: Configure Nginx SSL
+  sudo: yes
+  shell: bash ./configure_ssl.sh {{ NGINX_ENABLE_SSL }} {{ EDXAPP_LMS_SSL_NGINX_PORT }} {{ EDXAPP_CMS_SSL_NGINX_FRONTEND_PORT }}
+  args:
+    chdir: "{{ edxapp_files_path }}/nginx"
+  register: bash
+
+# Print the shell tasks's stdout and stderr.
+- debug: var="bash.stdout_lines + [ bash.stderr ]"
+
+- name: Restart Nginx to use the new settings
+  sudo: yes
+  command: service nginx restart
+
+#- name: Apply patch to authenticate user after their reset confirmation goes through
+#  sudo: yes
+#  shell: patch -p1 -d {{ edx_platform_path }} < {{ edxapp_files_path }}/patches/9b1f89d19ad26625859f887b12931f4d48dd3ff0_updated.patch
+#  register: patch_result
+#  ignore_errors: True
+
+# Print the results from patching.
+#- debug: var="patch_result.stdout_lines + [ patch_result.stderr ]"
+
+# Patch fails on stamp - re-enable once fixed
+#- name: Recompile python files touched by patching
+#  sudo: yes
+#  shell: python -m py_compile {{ forms_py }} {{ test_password_policy_py }} {{ test_reset_password_py }} {{ views_py }} {{ password_policy_validators_py }} {{ test_password_history_py }}
+#  register: compile_result
+#  when: patch_result|success
+
+# Print the results from compiling.
+#- debug: var="compile_result.stdout_lines + [ compile_result.stderr ]"
+#  when: patch_result|success
```

Comment:
note to other reviewers, the meaningful changes begins at line 46 (all additions). The rest is changing away from dos-style eol towards unix-style endings